modules/administration/pages/public-cloud.adoc

```diff
@@ -0,0 +1,351 @@
+|===
++
+The current value for the [replaceable]``Instance-ID`` or [replaceable]``Instance-Name`` in case of the Azure Cloud, can be obtained from the public cloud Web console or from within a terminal session as follows:
+** Obtain instance id from within Amazon EC2 instance
++
+
+----
+$ ec2metadata --instance-id
+----
+** Obtain instance id from within Google Compute Engine instance
++
+
+----
+$ gcemetadata --query instance --id
+----
+** Obtain instance name from within Microsoft Azure instance
++
+
+----
+$ azuremetadata --instance-name
+----
+
++
+After logging in through the {productname} Server {webui}, *change* the default password.
++
+{productname} Proxy does not have administration access to the {webui}.
+It can be managed through its parent {productname} Server.
+
+
+[[using-separate-storage-volume]]
+=== Using Separate Storage Volume
+
+
+We recommend that the repositories and the database for {productname} be stored on a virtual storage device.
+This best practice will avoid data loss in cases where the {productname} instance may need to be terminated.
+These steps *must* be performed *prior* to running the YaST {productname}  setup procedure.
+
+
+. Provision a disk device in the public cloud environment, refer to the cloud service provider documentation for detailed instructions. The size of the disk is dependent on the number of distributions and channels you intend to manage with {productname}.
+For sizing information refer to https://www.suse.com/support/kb/doc.php?id=7015050[SUSE Manager sizing examples]. A rule of thumb is 25 GB per distribution per channel.
+. Once attached the device appears as Unix device node in your instance. For the following command to work this device node name is required. In many cases the attached storage appears as **/dev/sdb**. In order to check which disk devices exists on your system, call the following command:
++
+
+----
+$ hwinfo --disk | grep -E "Device File:"
+----
+. With the device name at hand the process of re-linking the directories in the filesystem {productname} uses to store data is handled by the suma-storage script. In the following example we use [path]``/dev/sdb`` as the device name.
++
+
+----
+$ /usr/bin/suma-storage /dev/sdb
+----
++
+After the call all database and repository files used by SUSE Manager Server are moved to the newly created xfs based storage.
+In case your instance is a {productname} Proxy, the script will move the Squid cache, which caches the software packages, to the newly created storage.
+The xfs partition is mounted below the path [path]``/manager_storage``.
+.
+. Create an entry in /etc/fstab (optional)
++
+Different cloud frameworks treat the attachment of external storage devices differently at instance boot time.
+Please refer to the cloud environment documentation for guidance about the fstab entry.
++
+If your cloud framework recommends to add an fstab entry, add the following line to the */etc/fstab* file.
++
```

Comment:
The suma-setup script in the SUMa 4 images adds the fstab entry automatically so that paragraph should be removed.